.github/workflows/release.yml

```diff
@@ -4,46 +4,197 @@ on:
     tags:
       # this is a glob, not a regexp
       - '[0-9]*'
+
+
 jobs:
-  cockpituous:
+  source:
     runs-on: ubuntu-latest
-    environment: release
     container:
-      image: ghcr.io/cockpit-project/release
+      image: ghcr.io/cockpit-project/unit-tests
+      options: --user root
     permissions:
-      # create GitHub release
```

Comment:
I find it nice to justify the permissions with comments. Did you drop that on purpose or by accident?